skfp/descriptors/constitutional.py

```diff
@@ -0,0 +1,278 @@
+from rdkit.Chem import AddHs, Mol
+from rdkit.Chem.Descriptors import MolWt
+from rdkit.Chem.rdMolDescriptors import CalcNumRings, CalcNumRotatableBonds
+
+
+def atom_count(mol: Mol, atom_symbol: str) -> int:
+    """
+    Specific Atom Count.
+
+    Calculates the count of atoms [1]_ of a specific type (e.g., "C" for carbon, "H" for hydrogen).
```

Comment:
1. I don't think that reference here is necessary. This is arguably the simplest descriptor possible
2. Comma instead of parentheses: `type, e.g. "C" for carbon`.
3. Note that for hydrogens you should return total number of hydrogens, i.e. explicit and implicit, and this should be noted here